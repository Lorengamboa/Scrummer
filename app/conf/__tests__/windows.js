const { mainWindowSettings, createWindowSettings } = require('../windows');

test('mainWindowSettings', () => {
  expect(mainWindowSettings).toMatchSnapshot();
});

test('createWindowSettings', () => {
  expect(createWindowSettings).toMatchSnapshot();
});