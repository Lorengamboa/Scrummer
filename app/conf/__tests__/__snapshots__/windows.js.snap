// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createWindowSettings 1`] = `
Object {
  "height": 800,
  "resizable": false,
  "title": "New note",
  "width": 500,
}
`;

exports[`mainWindowSettings 1`] = `
Object {
  "webPreferences": Object {
    "backgroundThrottling": false,
  },
}
`;
